project/Build.scala

```diff
@@ -817,6 +817,74 @@ object Build {
       javaOptions := (javaOptions in `dotty-compiler-bootstrapped`).value
     )
 
+  lazy val `dotty-scala-library-from-tasty-tests` = project.in(file("dotty-scala-library-from-tasty-tests")).
```

Comment:
I don't understand what this project is for.

Reply:
It is there to test that we can recompile the jar from `dotty-scala-library` from a bootstrapped compiler with the normal bootstrapped library. We needed an extra project to get the dependencies on the bootstrapped compiler right.